express/properties/workflow.py

```diff
@@ -10,7 +10,7 @@ class WorkflowProperty(BaseProperty):
     Base class for workflow properties extracted in Express
     """
 
-    def __init__(self, name, parser, *args, **kwargs):
+    def __init__(self, name: str, parser: Any, *args, **kwargs):
```

Comment:
This is fine, but in general it's probably not adding much to type hint Any.

Reply:
Removed a few of the typehints for these function signatures